src/bun.js/bindings/webcore/JSEventEmitter.cpp

```diff
@@ -237,6 +237,15 @@ static inline JSC::EncodedJSValue addListener(JSC::JSGlobalObject* lexicalGlobal
     RETURN_IF_EXCEPTION(throwScope, encodedJSValue());
     auto result = JSValue::encode(toJS<IDLUndefined>(*lexicalGlobalObject, throwScope, [&]() -> decltype(auto) { return impl.addListenerForBindings(WTFMove(eventType), WTFMove(listener), once, prepend); }));
     RETURN_IF_EXCEPTION(throwScope, encodedJSValue());
+
+    JSC::Identifier newListenerEventType = JSC::Identifier::fromString(vm, "newListener"_s);
```

Comment:
future TODO: add fields for the super common properties like captureRejection, newListener, error, etc to avoid creating identifiers like this